firebase-ai/src/main/kotlin/com/google/firebase/ai/type/Content.kt

```diff
@@ -44,14 +44,17 @@ constructor(public val role: String? = "user", public val parts: List<Part>) {
   public class Builder {
```

Comment:
LGTM, but leaving this comment to prevent auto-merge. Could you add a changelog entry for this? Could just be like "added missing setters and fixed builders not returning themselves in java" or something akin.

Reply:
+1 update the api file. @daymxn this will need to land in `firebase-vertexai` too.